### PR TITLE
Update Style Guide for Logical Operators

### PR DESCRIPTION
**Pull request by arnavb**
**Fixes issue:** #3308 (Check the comments of @iattempt's review)

**Changes:**
- Update Style Guide for Logical Operators so that they are placed after their condition.

### CODE REVIEW

File: guides/coding_style/c++/README.md [hunk 1]
```diff
@@ -155,15 +156,15 @@ void foo()
...
         res &= true;
     else
         res &= false;
 
-    if (res != true
-        || (!(res == true)
-            || (res == false)))
+    if (res != true ||
+         (!(res == true) ||
+            (res == false)))
```
Comment (iattempt) [inline guides/coding_style/c++/README.md]: In my opinion, the conditions must be aligned to previous line.
i.e., in this situation, the code will be:
```
    if (res != true ||
        (!(res == true) ||
         (res == false)))
```

Comment (iattempt) [reply guides/coding_style/c++/README.md]: If there used parentheses then align with the char after open parenthesis.

**Comment by arnavb** [comment]
@AdiChat please merge this now!

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**